docker-compose.ci.yml

```diff
@@ -0,0 +1,71 @@
+version: '3'
+
+
+services:
```

Comment:
lots of duplication with `dev`, I would extract the common stuff into a `docker-compose.base.yml`, but PhpStorm doesn't support multiple docker-compose.yml files and I wanted to keep that route open